src/nyc_trees/apps/survey/survey_detail.sql

```diff
@@ -1,35 +1,17 @@
 -- Note: This was adapted into trees.sql.  Try to keep changes in sync
```

Comment:
We have diverged from `trees.sql`. We may way to expand the comment to explain that the logic is not different, just the source of the records.